### PR TITLE
Fix openshift-acme e2e

### PR DESCRIPTION
**Pull request by tnozicka**
Fixes: https://github.com/openshift/release/issues/1464

/cc @stevekuznetsov

### CODE REVIEW

File: ci-operator/config/tnozicka/openshift-acme/master.yaml [hunk 1]
```diff
@@ -14,10 +24,28 @@ images:
       as:
       - bin
   to: openshift-acme
+
 promotion:
   cluster: https://api.ci.openshift.org
   namespace: ci
   tag: latest
+
+tag_specification:
```
Comment (tnozicka) [inline ci-operator/config/tnozicka/openshift-acme/master.yaml]: Is adding this enough or it requires modifications of https://github.com/openshift/release/blob/2f38d36898dda73d24dae62d5e9b25dc186f5d0f/ci-operator/jobs/tnozicka/openshift-acme/manual-presubmits.yaml ?

**Comment by tnozicka** [comment]
so this wasn't enough, same error remains
https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/tnozicka_openshift-acme/79/pull-ci-openshift-acme-e2e/8
https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/tnozicka_openshift-acme/79/pull-ci-openshift-acme-e2e/9
ideas?

**Comment by bbguimaraes** [comment]
You need to configure `cluster/ci/config/prow/plugins.yaml` to update the `ci-operator-tnozicka-openshift-acme` configmap when the files are changed.

**Pull request merged**